File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/java/cwe-798/code_before1.java
// From cwe-snippets, snippets_2/non-compliant/Java/0170.java

private static String hmacKey = "";
byte[] keyBytes = hmacKey.getBytes();
SecretKeySpec key = new SecretKeySpec(keyBytes, "SHA1");
Mac hmac = Mac.getInstance("HmacSHA1");
hmac.init(key);

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/c/cwe-798/code_before1.c
// From cwe-snippets, cat ./snippets_1/non-compliant/C/0266.c

int VerifyAdmin(char *password) {
    if (strcmp(password,"68af404b513073584c4b6f22b6c63e6b")) {
                           printf("Incorrect Password!\n");
                           return(0);
    }
                       return(1);
}